### PR TITLE
Index book links

### PR DESCRIPTION
**Pull request by AndreLemay**
Link to books in index columns everywhere (except for books linking to themselves on their own page).

Would it be possible to put a halt on adding new data until this PR is completed? This PR changes all the index fields to using the book generatedId instead of the name, so any new data added will also have to be changed before this can be completed.

Also note that this depends on #118 and #74 being completed, as the pages will fail to load without that data.

### CODE REVIEW

**Comment by AndreLemay** [comment]
Cloud build is currently failing because of the issues mentioned in #118

**Comment by duffn** [comment]
I will hold on all data updates until after this.

The priest is on page 36. Do you want to update that in this PR?

I have no idea where the built-in comlink is from. Remove it for now so it's not blocking this and update the comlink issue to note that it needs adding back in at some point.

**Comment by duffn** [comment]
This PR looks great. 

I look forward to poking around at the branch build on cloud when the conflicts are resolve.

**Comment by AndreLemay** [comment]
Sure, I'll fix it. Just landed back home, so I should be able to get that stuff sorted out in a couple hours

**Comment by duffn** [comment]
As far as Codecov, we can ignore the checks for now. I may end up turning them off for the meantime.

**Comment by AndreLemay** [comment]
Yeah they're not super useful until we get some tests written

**Comment by duffn** [comment]
Okay, disabled.

File: src/data/talents.yaml [hunk 1]
```diff
@@ -1965,19 +1965,19 @@
   activation: Passive
   ranked: No
   forceSensitive: No
-  index: Dawen of Rebellion:115
```
Comment (duffn) [inline src/data/talents.yaml]: DAWEN!

**Comment by duffn** [comment]
Poking around the build and so far so good!

Two notes:
- Can we make the indices on individual cards links as well? But leave them the same style they are right now. Also those are just IDs and not names at the moment. 
- The generaredId shows in a book individual card.

**Comment by AndreLemay** [comment]
The only problem with leaving the current style is that there's no indication that they're links unless you happen to hover your cursor over them.

**Comment by duffn** [comment]
Yeah, let's see how that works.

**Comment by duffn** [comment]
Actually just the regular links are fine.

**Comment by AndreLemay** [comment]
Ok, I'll change those back and merge

**Pull request merged**